manifest_staging/charts/secrets-store-csi-driver/templates/csidriver.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: storage.k8s.io/v1beta1
+apiVersion: {{ template "csidriver.apiVersion" . }}
```

Comment:
Will `deploy/csidriver.yaml` be updated when a release is cut?

Reply:
good catch! maybe we need a new csidriver yaml similar to manifest_staging/deploy/csidriver-1.15.yaml for <1.18

Reply:
I was actually thinking of updating the `deploy/csidriver.yaml` a little later as `storage.k8s.io/v1beta1` isn't going to be deprecated until `1.22`. We'll need to maintain 3 YAMLS for csi driver otherwise:

1. `deploy/csidriver-1.15.yaml`
2.`deploy/csidriver-1.16-1.17.yaml`
3.`deploy/csidriver-1.18+.yaml`

Reply:
I'm fine with doing the change after 1.21 is released at which point it will be safe to remove 1.17 and earlier as they will not be supported in K8s upstream.

Reply:
Exactly! Opened an issue to track it: https://github.com/kubernetes-sigs/secrets-store-csi-driver/issues/403